jepsen/src/jepsen/nemesis/combined.clj

```diff
@@ -249,6 +263,133 @@
                              :fs    #{:strobe-clock}
                              :color "#A0E9E3"}}})))
 
+(defn wan-nemesis
+  "A nemesis which simulates running the cluster on a wide-area network, e.g.
+  a cluster of nodes split across multiple different AWS regions, i.e. higher
+  average latency, larger latency variance and spottier connectivity.
+
+  This nemesis responds to the following f's:
+
+      :drop     (Drop traffic between two nodes.)
```

Comment:
Should this be :start-partition?

Reply:
To be honest, I'm a bit confused about `:drop` and it's relation to the partitions nemesis. I've simply mirrored what's in `jepsen.net`. I guess `:drop` has a direction, i.e. `src` and `dest`?

Reply:
@Danten pointed out that partitions are also always splitting a cluster in two halves, whereas drop can be between two nodes.

I guess one could argue that that the partition nemesis should be made more general by including what `drop` does? But again this seems like a separate issue, `drop` is currently part of `jepsen.net` and that's what this nemesis uses in its `invoke!`...